### ISSUE TITLE
Alt text should be below the image, not above it

### ISSUE THREAD

**Issue by StefanLobbenmeier** [TzorAL/daily-random-xkcd#4]
To me the alt text usually refers to something in the comic so I first read the comic and then the alt text. So the visual order should reflect that too.

(I read this feed in freshrss in case it matters)
